docker/scripts/setup

```diff
@@ -29,7 +29,7 @@ if [ -d "/scripts/cache" ]; then
   mv /scripts/cache vendor/
   chown -R huginn:huginn vendor/cache
 fi
-sudo -u huginn -H bundle install --deployment --without development test
+sudo -u huginn -H bundle install --deployment --with deployment
```

Comment:
@dsander, if `sudo -u huginn -H bundle install --deployment --without test` fixes the issue for you, could you make that change and merge it?  I don't have Docker configured so I can't test.  (I should try and fix that this weekend.)